plasmapy/physics/parameters.py

```diff
@@ -1506,13 +1385,10 @@ def lower_hybrid_frequency(B, n_i, ion='p+'):
     except Exception:
         raise ValueError("Invalid ion in lower_hybrid_frequency.")
 
-    try:
-        omega_ci = gyrofrequency(B, particle=ion)
-        omega_pi = plasma_frequency(n_i, particle=ion)
-        omega_ce = gyrofrequency(B)
-        omega_lh = 1 / np.sqrt((omega_ci * omega_ce) ** -1 + omega_pi ** -2)
-        omega_lh = omega_lh.to(u.rad / u.s)
-    except Exception:
-        raise ValueError("Unable to find lower hybrid frequency.")
+    omega_ci = gyrofrequency(B, particle=ion)
+    omega_pi = plasma_frequency(n_i, particle=ion)
+    omega_ce = gyrofrequency(B)
+    omega_lh = 1 / np.sqrt((omega_ci * omega_ce) ** -1 + omega_pi ** -2)
```

Comment:
I think this line can be improved. Here we are using two different ways to get the reciprocal of a square root value. One is is `1 / np.sqrt()` and the other is `omega_pi ** -2`. In terms of readability I like `** -2` but I think `np.sqrt()` may be faster from an optimization perspective. We should check this.

Reply:
Yeah, but given that *premature optimization yadda yadda worst thing since goto*, I'm just going to do this with `**-0.5` and stick a TODO note in there for now.

Reply:
Hence the "we should check this" :smile: